README.md

```diff
@@ -43,6 +43,14 @@ export UAA_DEFAULT_USER_PW=YOUR_PASSWORD
 export VAULT_AUTO_UNSEAL=true
 ```
 
+In order to run Cloudbreak, Periscope and Datalake from IDEA, put this into your Profile:
+
+```
+export CB_LOCAL_DEV=true
+```
+
+Cloudbreak, Periscope and Datalake containers will be stopped and Uluwatu will connect to java processes running on your host.
```

Comment:
This is not true anymore. Those containers are not even started.

Reply:
fixed